.scalafmt.conf

```diff
@@ -20,5 +21,9 @@ project.excludeFilters = [
 fileOverride {
   "glob:**/scala-3*/**" {
     runner.dialect = scala3
+    rewrite.scala3.convertToNewSyntax = true
```

Comment:
I would for sure try to keep these too rules. Maybe redundant braces also outside the scala 3 codebase?